google/cloud/aiplatform/tensorboard/uploader_utils.py

```diff
@@ -81,6 +84,96 @@ def __init__(self, experiment_resource_name: str, api: TensorboardServiceClient)
         self._run_name_to_run_resource_name: Dict[str, str] = {}
         self._run_tag_name_to_time_series_name: Dict[(str, str), str] = {}
 
+    def batch_create_runs(self, run_names: List[str]) -> List[TensorboardRun]:
+        """
+       Batch creates TensorboardRuns.
+       :param run_names: a list of run_names for creating the TensorboardRuns.
+       :return: the created TensorboardRuns
+       """
+        i = 0
+        batch_size = OnePlatformResourceManager.CREATE_RUN_BATCH_SIZE
+        created_runs = []
+        while i * batch_size < len(run_names):
```

Comment:
Why not a for loop? The while loop makes it seems like the number of iterations is indefinite.